app/assets/javascripts/Components/Result/marks_panel.jsx

```diff
@@ -180,8 +180,9 @@ class CheckboxCriterionInput extends React.Component {
           <div>
             {!this.props.released_to_students &&
              <span className='checkbox-criterion-inputs'>
-              <label>
+              <label onClick={() => this.handleChange({target: {value: 'yes'}})}>
                 <input
+                  id={`checkbox_criterion_${this.props.id}_yes`}
```

Comment:
should I remove these IDs since we are not using them?

Reply:
Yes, I think you should remove them.